schemas/fingerprint-server-api.yaml

```diff
@@ -674,6 +696,29 @@ components:
           required:
             - code
             - message
+    ErrorCommon404Response:
```

Comment:
I think this should be called `ErrorVisits404Response`? Because the error code is specific to a non-existent visitor ID? Similiar how we have `ErrorEvent404Response` for a non-existent request ID ?

Reply:
But, the Related Visitors API also shares this response...still call it Visits?

Reply:
Ah yes, `ErrorVisitor404Response` ?